Assets/MRTK/Core/Services/BaseService.cs

```diff
@@ -24,13 +24,30 @@ public abstract class BaseService : IMixedRealityService
         public virtual BaseMixedRealityProfile ConfigurationProfile { get; protected set; } = null;
 
         /// <inheritdoc />
-        public virtual void Initialize() { }
+        public virtual void Initialize() 
+        {
+            if (IsMarkedDestroyed)
+            {
+                throw new InvalidOperationException("It is not allowed to re-initialize a service after Destroy() has been called.");
```

Comment:
Why not?

Reply:
the thought here was to disallow because Destroy implies that the object is going away. I could1 100% be wrong here.

Reply:
upon further review, removed the check